tests/lib/assertions/developer1-pi-uc20.model.json

```diff
@@ -10,12 +10,12 @@
     "base": "core20",
     "snaps": [
         {
-            "default-channel": "20-pi/edge",
+            "default-channel": "20/edge",
             "id": "YbGa9O3dAXl88YLI6Y1bGG74pwBxZyKg",
             "name": "pi",
             "type": "gadget"
         }, {
-            "default-channel": "20-pi3/edge",
+            "default-channel": "20/edge",
```

Comment:
@xnox Hm, I remember Gustavo suggested back in the UC18 days that we should have tracks per supported board. Do you know why this go changed and who changed it?